budgetApp1/budgetApp1/LoginViewController.swift

```diff
@@ -21,7 +22,6 @@ class LoginViewController: UIViewController {
             performSegue(withIdentifier: "loginSegue", sender: self)
         }
         else {
-//            let alert2 = Alert.authenticationAlert(on: UIViewController)
             let alertController = UIAlertController(
                             title: "Invalid credentials",
```

Comment:
Keep these strings in a constant file. Also, it's not really a swifty approach to throw an alert to say invalid credentials. Maybe use a more swift/ios approach 😄

Reply:
Will do.